source/_includes/header.html

```diff
@@ -24,7 +24,7 @@
               </a>
               <ul class="dropdown-menu">
                 <li><a href="/index.html">Welcome</a></li>
-                <li><a href="/downloads.html">Downloads</a></li>
+                <li><a href="/downloads.html">Releases</a></li>
```

Comment:
Usually the home page has a link named `Download` to latest release so I think it should be `<li><a href="/download">Download</a></li>` (keeps `Download` but changes `href`).

Reply:
👍 I second that.

Reply:
The should be one more menu item Download

Reply:
No! then usually I saw that that `Download` page instead has a link to `prior releases`.

Reply:
Well, there is now #56 to add one more item.

@yasserzamani Do you suggest to remove link to /releases ?
It is very good history page, with links to all release notes.

BTW, Struts 2.5.14  notes link to wrong Struts 2.5.13

Reply:
Download `/download` for link to current release that users needs to download.  
Releases `/downloads` all current and previous releases as releases history.

Reply:
I think usually it's not in home page so I prefer deleting it's link from home page then adding a link in `download.html` as below:

```diff
</p>
+
+<h2 id="all-releases">All releases</h2>
+<p>
+  If you are looking for previous versions of above please check <a href="downloads.html">Releases</a> page.
+</p>

<h2 id="prior-releases">Prior releases</h2>
```

Reply:
Sorry I did not see that there is already a `Download` button on home page outside of the menu. Then your current work is nice i.e. keep `Releases` link in menu. Thanks!